send_to_s3.py

```diff
@@ -8,35 +8,49 @@
 import os
 import datetime
 
-conn = boto.connect_s3();
-bucketname = sys.argv[1]
-bucket = conn.get_bucket(bucketname)
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
 
-buildname = sys.argv[2]
-sourcedir = sys.argv[3]
-destdir = sys.argv[4]
+try:
+    try:
+        conn = boto.connect_s3()
+    except Exception as e:
+        eprint("Error connecting to s3, trying again: {0}".format(e))
+        conn = boto.connect_s3()
```

Comment:
Yes, this will be very helpful.  I'm excited to see if this cleans up the s3 problem.  For reference, there is a branch that has now shown the s3 problem twice in a row [here](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-f28381f6f9bbaf99f4b8f366c4d59f7007f171d6-x86_64-Linux-Ubuntu-14.04-gcc-4.8.html) if you are interested in trying a specific branch or investigating.  I saw the s3 errors, deleted the build results, and it had the same result.  May be a repeatable test case.

Reply:
Sounds good, I'll merge this then try the rebuild in a few hours (to make sure it picks up the new version of decent_ci)